lib/gooddata/lcm/actions/collect_data_product.rb

```diff
@@ -0,0 +1,62 @@
+# encoding: UTF-8
+#
+# Copyright (c) 2010-2017 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+require_relative 'base_action'
+
+module GoodData
+  module LCM2
+    class CollectDataProduct < BaseAction
+      DESCRIPTION = 'Collect DataProduct'
+
+      PARAMS = define_params(self) do
```

Comment:
Identical code found in collect_data_product.rb:14, ensure_data_product.rb:14 (mass = 100)